assignments/prototypes.js

```diff
@@ -1,110 +1,122 @@
-/*
-  Object oriented design is commonly used in video games.  For this part of the assignment you will be implementing several constructor functions with their correct inheritance hierarchy.
+class GameObject {
+  constructor(att) {
+    this.createdAt = att.createdAt;
+    this.dimensions = att.dimensions;
+  }
+  destroy() {
+    console.log(`*** ${this.name} was destroyed. ***`); 
+  }
+}
```

Comment:
you came back in a refactored to class syntax, nice! I'm glad you were able to jump to the next logical step like that, it's also important that you were able to see how everything works under the hood with classes with functional prototypes like these were initially